ui/src/mocks/server-response.ts

```diff
@@ -6,7 +6,12 @@ export const mockServerResponse = {
         create: `/servicesNS/nobody/${MOCKED_TA_NAME}/${MOCKED_TA_INPUT}/_new`,
     },
     updated: '2023-08-21T11:54:12+00:00',
-    entry: [],
+    entry: [
+        {
+            name: 'name',
```

Comment:
This response is a generic response that is used as a fallback for stories and unit tests. I would rather to add it to the specific story. Otherwise we may break something (and [we did](https://github.com/splunk/addonfactory-ucc-generator/actions/runs/8891031418/job/24412163603?pr=1148))

Reply:
or you may fix tests to support this scenario

Reply:
I have added a new mock server response for the input page.